smartva/utils/utils.py

```diff
@@ -111,8 +111,10 @@ def int_or_float(x):
 def identity(arg):
     return arg
 
-
-def value_from_row(row, fn=identity, default=None):
+# in python 2, None < x is True for any float x, but in python 3 it is an error
+# so I have made the default value -inf instead of None
```

Comment:
There are good ways to test for None if that's what you need. I'm not sure where this is used in the logic, so not sure if that makes sense, but I prefer None for missing info to -inf.